evaluation/evaluate.py

```
@@ -111,6 +111,48 @@ def label_row(ws, row, label, value, label_fill=None):
+        ws1.merge_cells(f"A{start_row}:F{start_row}")
+        title_cell = ws1[f"A{start_row}"]
+        title_cell.value = "📊 질문 유형별 성공률"
+        title_cell.font = H_FONT
+        title_cell.fill = TEAL_FILL
+        title_cell.alignment = AL_C
+        ws1.row_dimensions[start_row].height = 22
+
+        # 테이블 헤더
+        cat_headers = ["질문 유형", "질문 수", "Vector 성공 수", "Vector 성공률", "Hybrid 성공 수", "Hybrid 성공률"]
+        cat_widths  = [18, 10, 14, 14, 14, 14]
+        for col_idx, (h, w) in enumerate(zip(cat_headers, cat_widths), start=1):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)